T8Suite/Form1.cs

```diff
@@ -2685,6 +2685,21 @@ private void Form1_Load(object sender, EventArgs e)
             SetupDocking();
             LoadMyMaps();
         }
+
+        private void ShowBtnIfSymbolInBin(string symbolname, DevExpress.XtraBars.BarButtonItem btn )
+        {
+            SymbolCollection sc = (SymbolCollection)gridControlSymbols.DataSource;
+            foreach (SymbolHelper sh in sc)
+            {
+                if (sh.SmartVarname == symbolname)
+                {
+                    btn.Visibility = BarItemVisibility.Always;
+                    return;
+                }
+            }
+            btn.Visibility = BarItemVisibility.Never;
+        }
+
```

Comment:
@mattiasclaesson @MelvisR I was looking over the DynamicTuningMenu() and I was thinking if it would make sense to use something like this where we check if the symbol is in the bin and then show/hide the button(s) based on that

Reply:
the usage for the function https://github.com/mattiasclaesson/TuningSuites/pull/48/files#diff-e8c3387c5140840ac13b246cfe544f27f26a2605ca7c52871e81e648fa75185dR2787-R2791